admin_ui_support/src/Plugin/rest/resource/SimpleConfigResource.php

```diff
@@ -124,11 +127,31 @@ public function get() {
    */
   public function patch(array $data) {
     $config = $this->configFactory->getEditable($this->getPluginDefinition()['config_name']);
+    $this->handleNewConfig($config);
     $config->setData($data);
     $config->save();
     return new ModifiedResourceResponse($config);
   }
 
+  /**
+   * Responds to POST requests.
+   *
+   * @param array $data
+   *   The unserialized data.
+   *
+   * @return \Drupal\rest\ModifiedResourceResponse
+   *   The modified resource response.
+   */
+  public function post(array $data) {
```

Comment:
Does POST really make sense for simple configuration?

Reply:
Ok will remove